remotehelper/runner.go

```diff
@@ -104,18 +104,11 @@ func (r *Runner) Run(ctx context.Context, remoteName string, remoteUrl string) e
 		case "list":
 			refs, err := remote.List(&git.ListOptions{})
 
-			if err == transport.ErrRepositoryNotFound && args == "for-push" {
```

Comment:
This used to auto-create a repo when pushing to dgit for the first time, now it instructs them to run `dgit init`